[2024-08-22T14:50:09.559+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: sarima_forecast_dag.run_sarima_forecast manual__2024-08-22T14:49:16.739567+00:00 [queued]>
[2024-08-22T14:50:09.566+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: sarima_forecast_dag.run_sarima_forecast manual__2024-08-22T14:49:16.739567+00:00 [queued]>
[2024-08-22T14:50:09.566+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-08-22T14:50:09.566+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-08-22T14:50:09.566+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-08-22T14:50:09.576+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): run_sarima_forecast> on 2024-08-22 14:49:16.739567+00:00
[2024-08-22T14:50:09.582+0000] {standard_task_runner.py:55} INFO - Started process 615 to run task
[2024-08-22T14:50:09.584+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'sarima_forecast_dag', 'run_sarima_forecast', 'manual__2024-08-22T14:49:16.739567+00:00', '--job-id', '10775', '--raw', '--subdir', 'DAGS_FOLDER/dag_forecast_ts_rainfall.py', '--cfg-path', '/tmp/tmpvccmbbfg']
[2024-08-22T14:50:09.585+0000] {standard_task_runner.py:83} INFO - Job 10775: Subtask run_sarima_forecast
[2024-08-22T14:50:09.628+0000] {task_command.py:388} INFO - Running <TaskInstance: sarima_forecast_dag.run_sarima_forecast manual__2024-08-22T14:49:16.739567+00:00 [running]> on host f64a695c5a62
[2024-08-22T14:50:09.677+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sarima_forecast_dag
AIRFLOW_CTX_TASK_ID=run_sarima_forecast
AIRFLOW_CTX_EXECUTION_DATE=2024-08-22T14:49:16.739567+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-22T14:49:16.739567+00:00
[2024-08-22T14:50:10.379+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_forecast_ts_rainfall.py", line 18, in run_sarima_forecast
    previsoes = sarima_plugin.forecast(n_periods=1)
  File "/opt/airflow/plugins/forecast_ts_rainfall_plugin.py", line 27, in forecast
    forecast = self.sarima_model.predict(n_periods=1)[0]
TypeError: predict() missing 1 required positional argument: 'params'
[2024-08-22T14:50:10.388+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=sarima_forecast_dag, task_id=run_sarima_forecast, execution_date=20240822T144916, start_date=20240822T145009, end_date=20240822T145010
[2024-08-22T14:50:10.403+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 10775 for task run_sarima_forecast (predict() missing 1 required positional argument: 'params'; 615)
[2024-08-22T14:50:10.438+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-08-22T14:50:10.450+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
